### PR TITLE
feat: Token List

### PR DESCRIPTION
**Pull request by Chef-Cheems**
This is kind of a draft cause there are some details to discuss in Slack.

### CODE REVIEW

**Review by hachiojidev — APPROVED** [review]

File: packages/token-lists/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "pancakeswap-token-lists",
```
Comment (RabbitDoge) [inline packages/token-lists/package.json:2]: `@pancakeswap/tokens-lists`

File: packages/token-lists/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "2.10.0",
```
Comment (RabbitDoge) [inline packages/token-lists/package.json:3]: v1 ?

File: packages/token-lists/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "@rollup/plugin-typescript": "^8.2.1",
+    "@types/jest": "^26.0.23",
+    "@types/node": "^15.0.2",
+    "babel-jest": "^26.6.3",
+    "jest": "^26.6.3",
+    "rollup": "^2.47.0",
+    "ts-jest": "^26.5.6"
+  },
+  "dependencies": {
+    "@uniswap/token-lists": "^1.0.0-beta.21",
```
Comment (RabbitDoge) [inline packages/token-lists/package.json:26]: do we need this ?

File: packages/token-lists/package.json (hunk 2 above)
Comment (Chef-Cheems) [reply packages/token-lists/package.json:3]: @RabbitDoge this is the version of the list we have right now in the exchange

File: packages/token-lists/package.json (hunk 3 above)
Comment (Chef-Cheems) [reply packages/token-lists/package.json:26]: This provides validation schema. We could of course copy it here 😅

File: packages/token-lists/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import buildList from "./buildList";
+
+console.log(JSON.stringify(buildList(), null, 2));
```
Comment (RabbitDoge) [inline packages/token-lists/src/index.ts:3]: rather use `.info` the user of `.log` is disabled in all our codebase (for debug only)

**Review by hachiojidev — APPROVED** [review]

**Review by RabbitDoge — APPROVED** [review]

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**